codecept.conf.js

```diff
@@ -0,0 +1,55 @@
+exports.config = {
+  output: './output',
+  helpers: {
+    Playwright: {
+      browser: 'chromium',
+      url: 'https://www.saucedemo.com',
+      show: false,
+    },
+  },
+  include: {
+    I: './steps_file.js',
+    loginPagePage: './bdd_tests/pages/LoginPage.js',
```

Comment:
```suggestion
    loginPage: './bdd_tests/pages/LoginPage.js',
```
Если примените, после нужно не забыть поменять и в тестах